.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - alpha
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: current
+      - uses: pnpm/action-setup@v2
```

Comment:
I suggest either all steps have `name` or only the ones that are 'user controlled'. A bit weird to me the two steps above that will pretty much never fail (and if they do there's nothing we can do about it) are named, but this step that could doesn't have a name. 🙈

Reply:
Thanks! We forgot about the name and although those are not user controller, we felt a name helps understand what is happening step by step in the workflow. [d3091ee](https://github.com/warp-ds/core/pull/3/commits/d3091ee6951ef423d4c8504f5b59a2c65dc6f9c9)